airplay/start.sh

```diff
@@ -10,17 +10,20 @@ if [[ -z $DISABLE_MULTI_ROOM ]] && [[ $CLIENT_ONLY_MULTI_ROOM == "1" ]]; then
   exit 0
 fi
 
-
 # Set the device broadcast name for AirPlay
 if [[ -z "$DEVICE_NAME" ]]; then
   DEVICE_NAME=$(printf "balenaSound Airplay %s" $(hostname | cut -c -4))
 fi
 
+if [[ "$BLUETOOTH_DEVICE_NAME" ]]; then
```

Comment:
That was fast, awesome work! I'm super nitpicking here... but if a user has both variables set, then `BLUETOOTH_DEVICE_NAME` will end up being used. I think it's better if we stick with `DEVICE_NAME` in that case. Something like:

```
if [[ -z "$DEVICE_NAME" ]]; then
   if [[ "$BLUETOOTH_DEVICE_NAME" ]]; then
     DEVICE_NAME="$BLUETOOTH_DEVICE_NAME"
   else
     DEVICE_NAME=$(printf "balenaSound Airplay %s" $(hostname | cut -c -4))
   fi
fi
```